chapters/introduction.tex

```diff
@@ -167,3 +167,5 @@ \section{Notation}\label{notation}
     der "(" SOME-TOKEN ")"
     | "(" differentiated-expression "+" differentiated-expression ")"
 \end{lstlisting}
+
+Annotations are described in \cref{annotations} and use a special syntax according to \cref{how-to-interpret-annotation-definitions}.
```

Comment:
There are other places with annotations as well, so better to not point at a specific chapter (and we the remaining link still goes to that chapter):
```suggestion
Annotations are defined using the syntactic forms of Modelica record definitions and component declarations, but with special semantics given in \cref{how-to-interpret-annotation-definitions}.
```